challenges/objects-arrays.js

```diff
@@ -14,107 +14,157 @@
+// {"animal_name":"White-cheeked pintail","population":1,"scientific_name":"Anas bahamensis","state":"Oregon"},
+// {"animal_name":"Black-backed jackal","population":2,"scientific_name":"Canis mesomelas","state":"Washington"},
+// {"animal_name":"Brolga crane","population":9,"scientific_name":"Grus rubicundus","state":"New Mexico"},
+// {"animal_name":"Common melba finch","population":5,"scientific_name":"Pytilia melba","state":"Pennsylvania"},
+// {"animal_name":"Pampa gray fox","population":10,"scientific_name":"Pseudalopex gymnocercus","state":"Connecticut"},
+// {"animal_name":"Hawk-eagle, crowned","population":10,"scientific_name":"Spizaetus coronatus","state":"Florida"},
+// {"animal_name":"Australian pelican","population":5,"scientific_name":"Pelecanus conspicillatus","state":"West Virginia"}];
 
-The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
+// /* Request 1: .forEach()
 
-*/
-const animalNames = [];
-console.log(animalNames);
+// The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
-/* Request 2: .map()    
+// */
+// const animalNames = [];
+// zooAnimals.forEach(function(element) {
+// animalNames.push(`${element.animal_name} ${element.scientific_name}`)
+// });
+// console.log(animalNames);
 
-The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
+// /* Request 2: .map()    
 
-*/
+// The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
-const lowerCase = [];
-console.log(lowerCase); 
+// */
 
-/* Request 3: .filter() 
+// const lowerCase = [];
+// zooAnimals.map(function(element) {
+// lowerCase.push(`${element.animal_name.toLowerCase()}`)
+// });
+// console.log(lowerCase); 
 
-The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
+// /* Request 3: .filter() 
 
-*/
-const largerPopulation = [];
-console.log(largerPopulation);
+// The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
-/* Request 4: .reduce() 
+// */
+// const largerPopulation = [];
+// largerPopulation.push(zooAnimals.filter( currentValue => 
+//    currentValue.population < 5
+//    ));
```

Comment:
i notice you used an arrow function in this one, great job. you could do that in the previous ones as well. depends on what syntax you prefer, they both work